src/gpuocean/SWEsimulators/OceanStateNoise.py

```diff
@@ -455,6 +461,7 @@ def perturbOceanState(self, eta, hu, hv, H, f, beta=0.0, g=9.81,
 
                                                               np.float32(g), np.float32(f),
                                                               np.float32(beta), np.float32(y0_reference_cell),
+                                                              np.float32(0.0), np.float32(1.0), np.float32(0.0), np.float32(1.0),
```

Comment:
Are these correct? Can't find any matching parameters in ocean_noise.cu... @havahol